bin/live/pycbc_live_combine_single_significance_fits

```diff
@@ -193,7 +193,15 @@ for ifo in args.ifos:
 
         invalphan = c[valid] / a[valid]
         mean_alpha = c[valid].mean() / invalphan.mean()
-        cons_alpha = np.percentile(a[valid], 100 - args.conservative_percentile)
+        cons_count = np.percentile(
+            c[valid],
+            args.conservative_percentile
+        )
+        cons_invalphan = np.percentile(
+            invalphan,
+            args.conservative_percentile
+        )
+        cons_alpha = cons_count / cons_invalphan
         cons_alphas_out[ifo][counter] = cons_alpha
```

Comment:
Use the line for a comment rather than defining a variable which is only used once
```suggestion
        # Conservative alpha estimate using percentile rather than mean
        cons_alphas_out[ifo][counter] = cons_count / cons_invalphan 
```